src/components/Slider/index.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import Slider from '.';
+
+const ARROW_LEFT_KEYCODE = 37;
+const ARROW_RIGHT_KEYCODE = 39;
+
+describe('Slider', () => {
+  describe('given labels', () => {
+    test('renders slider with label for each step', () => {
```

Comment:
Why don't you use `it` instead?
```suggestion
    it('renders the slider with labels for each step', () => {
```

Reply:
My bad, I get used to using `test` from Elixir testing 🙈 Updated in [e774410](https://github.com/hanam1ni/react-surveyor/pull/100/commits/e774410d372f878135fe4812da5913aa27098586)